tasks/components/Input.tsx

```diff
@@ -22,7 +22,7 @@ type InputProps = {
    * @default noop
    */
   onChange?: (text: string) => void
-} & Omit<InputHTMLAttributes<Element>, 'id' | 'value' | 'label' | 'type' | 'onChange'>
+} & Omit<InputHTMLAttributes<HTMLInputElement>, 'id' | 'value' | 'label' | 'type' | 'onChange' | 'crossOrigin'>
```

Comment:
I'm not sure why `'crossOrigin'` was added here?

Reply:
I am also not sure

Reply:
I'll investigate the CI / Docker issue (later today)

Reply:
The docker setup seems to be a bit messy right now, pretty much all your changes look good but I might need to do some clean up when it comes to docker in order to resolve this

Reply:
Can you list your change requests, then someone else can take them over if necessary. Please open a new issue, not in the scope of this PR.